tests/unit_tests/levin.cpp

```diff
@@ -1017,7 +1154,7 @@ TEST_F(levin_notify, private_fluff_without_padding)
             auto notification = receiver_.get_notification<cryptonote::NOTIFY_NEW_TRANSACTIONS>().second;
             EXPECT_EQ(txs, notification.txs);
             EXPECT_TRUE(notification._.empty());
-            EXPECT_FALSE(notification.dandelionpp_fluff);
+            EXPECT_TRUE(notification.dandelionpp_fluff);
```

Comment:
Changed within this PR (see other comment within this review in `levin_notify.cpp`). All similar changes were for same reason.